src/commands/commandui.cpp

```diff
@@ -407,7 +409,8 @@ int CommandUI::run(QStringList& tokens) {
         engine, &QQmlApplicationEngine::objectCreated, qApp,
         [url](QObject* obj, const QUrl& objUrl) {
           if (!obj && url == objUrl) {
-            QGuiApplication::exit(-1);
+            logger.error() << "Failed to load " << objUrl.toString();
+            // QGuiApplication::exit(-1);
```

Comment:
why this?

Reply:
Accidental commit.  Sorry.  Was useful for QML debugging not to exit there.